app/views/app/index.html.haml

```diff
@@ -1,2 +1,4 @@
-= react_component 'ScreensmartApp', {csrfToken: form_authenticity_token},
-                                    class: 'app'
+:javascript
+  window.csrfToken = '#{form_authenticity_token}'
```

Comment:
Het is inderdaad de bedoeling om straks een dergelijke minimalistische library te gebruiken, maar de meeste moderne JS libraries hebben geen `npm`-loze distributie. En ik heb geen zin om een enorme hoeveelheid kleine dependencies in `vendor/assets/javascripts` te onderhouden. Vandaar de keuze om nu tijdelijk jQuery te gebruiken totdat we over zijn op npm voor de client